dashboards/eigenda-metrics.json

```diff
@@ -597,20 +630,16 @@
             "type": "prometheus",
             "uid": "prometheus"
           },
-          "disableTextWrap": false,
           "editorMode": "code",
-          "expr": "sum by(job) (node_eigenda_current_batches{type=\"size\"})\n/\nsum by(job) (node_eigenda_current_batches{type=\"number\"})",
-          "fullMetaSearch": false,
+          "expr": "sum(node_eigenda_processed_batches_total{type=\"size\"})\n/\nsum(node_eigenda_processed_batches_total{type=\"number\"})",
```

Comment:
The screenshoot shows this is trending upwards, but should it relatively stable since it's avg batch size (in bytes)?

Reply:
yea even I had this doubt but the scale if you see few KB. you think that's not right?

Reply:
this is the query
```
sum(node_eigenda_processed_batches_total{type="size"})
/
sum(node_eigenda_processed_batches_total{type="number"})
```

at any instant - it is doing sub(size) / sum(number)

Reply:
I also think it will stabilize over time since right now size is increasing more than number in terms of ratio of them so average would increase

Reply:
Ok, it looks trending upward just because the window is too short? like if it's over a few days, it's stable

Reply:
yea. updated the screenshot.